tools/testinfra/runner/src/main.rs

```diff
@@ -153,7 +207,13 @@ fn report<P: AsRef<Path>>(tests: Vec<TestResult>, path: P) -> Result<()> {
     })?;
     let mut xml = BufWriter::new(&file);
 
-    let failures = tests.iter().filter(|test| !test.passed).count();
+    let failures = tests
```

Comment:
This doesn't seem to be used?

Reply:
It is used when writing `<testsuites tests="{}" failures="{}">` <- here